envergo/templates/moulinette/_form_footer.html

```diff
@@ -5,7 +5,7 @@
     <h4 class="fr-accordion__title">
       <button class="fr-accordion__btn"
               type="button"
-              aria-expanded="false"
+              aria-expanded="{% if expand_optional_forms %}true{% else %}false{% endif %}"
```

Comment:
Je suggérerais bien [l'usage de yesno](https://docs.djangoproject.com/en/dev/ref/templates/builtins/#yesno).

```suggestion
              aria-expanded="{{ expand_optional_forms|yesno:"true,false" }}"
```